whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/NrtmVersionDao.java

```diff
@@ -35,7 +36,7 @@ public NrtmVersionDao(@Qualifier("nrtmDataSource") final DataSource dataSource)
     public Optional<VersionInformation> findLastVersion(final NrtmSource source) {
         try {
```

Comment:
minor: feel free to ignore. May be rename to VersionInfo ?

Reply:
there's already a VersionInfo in common, which the nrtm module depends on. keep this on the backburner, though -- VersionInformation is quite long and not all that descriptive. there's probably a better name for this class (NrtmPublishableDocumentVersion? nope, still too long. but you get the idea).